test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -298,12 +287,11 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
             helper.Process(context, output);
 
             // Assert
-            Assert.Null(output.TagName);
```

Comment:
I might have misunderstood but I thought that the whole point of the test was to check if the tag helper runs beyond the check for required attributes and not to test the entire behavior including all special cases.